hx711_rpio.go

```diff
@@ -0,0 +1,132 @@
+// +build !windows,gpiomem
+
+package hx711
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/stianeikeland/go-rpio/v4"
+)
+
+// HostInit opens /dev/gpiomem. This needs to be done before Hx711 can be used.
+func HostInit() error {
+	return rpio.Open()
+}
+
+// NewHx711 creates new Hx711.
+// Make sure to set clockPinNum and dataPinNum to the correct pins.
+// The pin numbers must comply with BCM numbering schema.
+// https://cdn.sparkfun.com/datasheets/Sensors/ForceFlex/hx711_english.pdf
+// https://godoc.org/github.com/stianeikeland/go-rpio#Pin
+func NewHx711(clockPinStr string, dataPinStr string) (*Hx711, error) {
```

Comment:
Was there a reason why you changed this back to Str from Num?

Reply:
Compilation :) I forgot to change the names of variables and with the build tags I missed this.

Should be nice to setup travis build that could build everything with all tags, and may be even run some tests.

Reply:
Still showing Str

Reply:
Yeah, fixed it.